<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditConstraintDialog</class>
 <widget class="QDialog" name="EditConstraintDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Constraint</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Variant A</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Address A</string>
       </property>
       <property name="buddy">
        <cstring>lineEditAddressA</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditAddressA">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>adsfasdf</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Variant B</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxVariantB">
       <item>
        <property name="text">
         <string>USA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EU</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DEMO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DEMO_JP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CUSTOM</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Author</string>
       </property>
       <property name="buddy">
        <cstring>lineEditAuthor</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditAuthor">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>asdfasdf</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Note</string>
       </property>
       <property name="buddy">
        <cstring>plainTextEditNote</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPlainTextEdit" name="plainTextEditNote">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string>asdfasdfasdf</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Address B</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditAddressB"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxVariantA">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>USA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EU</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DEMO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DEMO_JP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CUSTOM</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Certainty</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Enabled</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBoxCertainty"/>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="checkBoxEnabled">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxVariantA</tabstop>
  <tabstop>lineEditAddressA</tabstop>
  <tabstop>comboBoxVariantB</tabstop>
  <tabstop>lineEditAddressB</tabstop>
  <tabstop>spinBoxCertainty</tabstop>
  <tabstop>lineEditAuthor</tabstop>
  <tabstop>plainTextEditNote</tabstop>
  <tabstop>checkBoxEnabled</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditConstraintDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditConstraintDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
